merkato/merkato.py

```diff
@@ -66,7 +67,8 @@ def rebalance_orders(self, new_txes):
         # new_history is an array of transactions
         # new_txes is the number of new transactions contained in new_history
         self._debug(2, "merkato.rebalance_orders")
-        ordered_transactions = reversed(new_txes)
+        ordered_transactions = new_txes
+        print('ordered transactions rebalanced', ordered_transactions)
```

Comment:
Can we make all development print statements into self._debug(50,...) or higher? That way when we're all working we can set to debug level 49 to see normal output if we are looking at other stuff. @livinginformation @15chrjef

Reply:
I'm okay with that.